rstudio-tidyverse/rockcraft.yaml

```
@@ -0,0 +1,261 @@
+      curl -fsSL "https://github.com/conda-forge/miniforge/releases/download/${MINIFORGE_VERSION}/Miniforge3-${MINIFORGE_VERSION}-Linux-${MINIFORGE_ARCH}.sh.sha256" -o /tmp/Miniforge3.sh.sha256 
+      echo "$(cat /tmp/Miniforge3.sh.sha256 | awk '{ print $1; }')  /tmp/Miniforge3.sh" | sha256sum -c - 
+      rm /tmp/Miniforge3.sh.sha256 
+      /bin/sh /tmp/Miniforge3.sh -b -f -p "${CONDA_DIR}"
+      rm /tmp/Miniforge3.sh 
+      "${CONDA_BIN}/conda" config --system --set auto_update_conda false 
+      "${CONDA_BIN}/conda" config --system --set show_channel_urls true 
+      echo "python ==${PYTHON_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned 
+      "${CONDA_BIN}/conda" install -y -q \
+        python="${PYTHON_VERSION}" \
+        conda="${MINIFORGE_VERSION:0:-2}" \
+        pip="${PIP_VERSION}"           
+      "${CONDA_BIN}/conda" update -y -q --all 
+      "${CONDA_BIN}/conda" clean -a -f -y
+      
+      # install - r packages
+      echo "r-base ==${R_BASE_VERSION}" >> "${CONDA_DIR}/conda-meta/pinned" 
+      "${CONDA_BIN}/conda" install -y -q \
+        r-base="${R_BASE_VERSION}" \
+        r-png="${R_PNG_VERSION}" \
+        r-reticulate="${R_RETICULATE_VERSION}" \
+        r-shiny="${R_SHINY_VERSION}"     
+      "${CONDA_BIN}/conda" clean -a -f -y
+      
+      # R needs TZ set
+      echo "TZ=${TZ}" >> "${R_HOME}/etc/Renviron.site"
+      
+      # install - rstudio-server
+      curl -fsSL "${RSTUDIO_DEB_URL}" -o /tmp/rstudio-server.deb 
+      
+      # validate the build signature
+      gpg --keyserver keys.gnupg.net --keyserver pgp.surfnet.nl --recv-keys 3F32EE77E331692F 
+      gpg --keyserver keys.openpgp.org --recv-keys 51C0B5BB19F92D60 
+      dpkg-sig --verify /tmp/rstudio-server.deb 
+      rm -rf "${GNUPGHOME}" 
+      
+      # install rstudio-server
+      dpkg -i /tmp/rstudio-server.deb 
+      rm -f /tmp/rstudio-server.deb 
+                      
+      # Install - r-tidyverse                  
+      "${CONDA_BIN}/conda" install -y -q \
+        r-tidyverse="${R_TIDYVERSE_VERSION}" 
+                      
+      # Clean                  
+      "${CONDA_BIN}/conda" clean -a -f -y      
+      
+      # Create some directories for staging files
+      mkdir -p "${CRAFT_PART_INSTALL}/${CONDA_DIR}" \
+      "${CRAFT_PART_INSTALL}/etc" \
+      "${CRAFT_PART_INSTALL}/run" \
+      "${CRAFT_PART_INSTALL}/${HOME}" \
+      "${CRAFT_PART_INSTALL}/${HOME_TMP}" 
+
+      # Write some config files for activating conda in user sessions
+      chmod 2775 "${CRAFT_PART_INSTALL}/${CONDA_DIR}"
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${HOME}/.bashrc"
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${CRAFT_PART_INSTALL}/etc/profile"
+      echo "conda activate base" >> "${HOME}/.bashrc"
+      
+      # Do some cleanup
+      rm -rf "${HOME}/.cache"
+
+      # Stage the directories
+      cp -r "${HOME}" "${CRAFT_PART_INSTALL}/home"
```

issue: We want to move the files under to `jovyan`'s home
```suggestion
      cp -r "${HOME}" "${CRAFT_PART_INSTALL}/home/jovyan"
```

---

Fixed